### ISSUE TITLE
contacts keep getting moved back to Facebook Non-Friends group

### ISSUE THREAD

**Issue by brianjmurrell** [dequis/purple-facebook#330]
So, I don't use Facebook.  I only have an account there because the "no account" mode (where you just sign up with a telephone number and don't open a FB account) didn't work with Pidgin.  Even as such I chat successfully on Messenger (using this Pidgin plugin) without doing the whole "friend me" thing on FB.  So I don't actually have any FB "friends".

That results however, in my contacts continually being moved back to the Facebook Non-Friends group in Pidgin, even after I have (repeatedly) moved them to other groups.

Once I move a contact to a group of my choosing, it should remain there.

**Comment by EionRobb** [comment]
The Bonjour plugin used to have the same problem until it didn't https://developer.pidgin.im/ticket/5753

**Comment by brianjmurrell** [comment]
Indeed.  I wonder if the plugin is making the same kind of mistake.

**Comment by felagund** [comment]
Yeah, I am bitten by this with people who are not my friends byt that I message with repeatadly.

**Comment by brianjmurrell** [comment]
> Is this the same as #154?

Maybe at some level.  I left a comment there.  Either way, not being able to move contacts to a group and have them stay there is annoying at best.

**Comment by dequis** [comment]
Please try 0.9.5

https://github.com/dequis/purple-facebook/releases/tag/v0.9.5-9ff9acf9fa14

**Comment by brianjmurrell** [comment]
I have 0.9.5 installed.  Too early to know if contacts are going to get moved back to the _Facebook Non-Friends_ group, but one thing that I can say right away that is annoying is that even when those groups are empty (because I have moved buddies to other groups), and I remove them, they come back after a minute or two.

I realise I could toggle the _Buddies->Show->Empty Groups_ off but I have some groups, which can be empty that I want to see.

I simply want to be able to remove groups that are of no use to me.

If the group comes back because a buddy shows up that has never been moved to another group, that is fine, but when I remove it after moving said buddy, it should stay removed.

**Comment by brianjmurrell** [comment]
This problem seems to still be occurring with 0.9.5-1.9ff9acf9fa14.
